html/semantics/forms/the-input-element/hidden-charset-case-sensitive.html

```diff
@@ -19,10 +19,10 @@
   addEventListener("message", this.step_func_done(event => {
     const params = new URL(event.data).searchParams;
 
-    assert_true(params.get("_charset_") != "", "lowercase valid");
-    assert_true(params.get("_CHARSET_") != "", "uppercase valid");
-    assert_true(params.get("_ChArSeT_") != "", "mixed case invalid");
-    assert_true(params.get("_charſet_") == "", "non-ASCII invalid");
+    assert_not_equals(params.get("_charset_"), "", "lowercase valid");
+    assert_not_equals(params.get("_CHARSET_"), "", "uppercase valid");
+    assert_not_equals(params.get("_ChArSeT_"), "", "mixed case invalid");
+    assert_equals(params.get("_charſet_"), "", "non-ASCII invalid");
```

Comment:
I think the idea was to change them from not_equals to actually testing `assert_equals` against the correct value. Which I think is "UTF-8" in this case? Or maybe "utf-8"?

Reply:
ahh that makes more sense, done!